commit 363ca5a413c8b0377831b473526afb82a882380d (refs/changes/05/1505/1)
Author: David Pacheco <dap@joyent.com>
Date:   2017-02-14T11:21:39-08:00 (2 years, 8 months ago)
    
    joyent/mdb_v8#85 missing dependency for mdb_v8_version.c leads to build failure

diff --git a/GNUmakefile b/GNUmakefile
index 173f325..21453d3 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -165,9 +165,12 @@ include Makefile.arch.targ
 # is baked into the binary itself (so the dmod can report its version) and also
 # used in the publish target.
 #
-$(MDBV8_BUILD)/mdb_v8_version.c: version
+$(MDBV8_BUILD)/mdb_v8_version.c: version | $(MDBV8_BUILD)
 	tools/mkversion < $^ > $@
 
+$(MDBV8_BUILD):
+	$(MKDIRP)
+
 #
 # Include common Joyent Makefile for JavaScript "check" targets.
 #
